include/ignition/sensors/ImuSensor.hh

```diff
@@ -135,6 +135,12 @@ namespace ignition
       /// \brief Get the gravity vector
       /// \return Gravity vectory in meters per second squared.
       public: math::Vector3d Gravity() const;
+      
+      /// \brief Convey worl frame's orientation and heading offset
```

Comment:
Specify the rotation offset of the coordinates of the World frame relative to a geo-referenced frame

Reply:
Done !